webservices/schemas.py

```diff
@@ -18,6 +20,34 @@
 spec.definition('OffsetInfo', schema=paging_schemas.OffsetInfoSchema)
 spec.definition('SeekInfo', schema=paging_schemas.SeekInfoSchema)
 
+# A namedtuple used to help capture any additional columns that should be
+# included with exported data:
+# field: the field object definining the relationship on a model, e.g.,
+#   models.ScheduleA.committee (an object)
+# column: the column object found in the related model, e.g.,
+#   models.CommitteeHistory.name (an object)
+# label: the label to use for the column in the query that will appear in the
+# header row of the output, e.g.,
+#   'committee_name' (a string)
+# position: the spot within the list of columns that this should be inserted
+# at; defaults to -1 (end of the list), e.g.,
+#   1 (an integer, in this case the second spot in a list)
+
+# Usage:  Define a custom attribute in a schema's Meta options object called
+# 'relationships' and set to a list of one or more relationships.
+#
+# Note:  There is no clean way to provide default values for a namedtuple at
+# the moment. This wrapper is modeled after the following post:
+# https://ceasarjames.wordpress.com/2012/03/19/how-to-use-default-arguments-with-namedtuple/
+class Relationship(namedtuple('Relationship', 'field column label position')):
```

Comment:
Since the schemas include the model it would seem to reason that it would recognize that relationship method defined on the model, where this seems duplicative.  But we discussed it and it seemed like you tried to get the downloads to work that way.  That's really the only thing I'm iffy on.